modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -492,17 +489,52 @@ void cv::gimpl::ie::GIEExecutable::run(std::vector<InObj>  &&input_objs,
 namespace cv {
 namespace gimpl {
 namespace ie {
+inline int getIdxByName(const std::vector<std::string>& reshape_names, const std::string& name) {
+    const auto it = std::find(reshape_names.begin(), reshape_names.end(), name);
+    GAPI_Assert(it != reshape_names.end());
+    return std::distance(reshape_names.begin(), it);
+}
 
-static void configureInputInfo(const IE::InputInfo::Ptr& ii, const cv::GMetaArg mm) {
+inline void configureReshape(const IE::InputInfo::Ptr& ii, IEUnit* const uu, const std::vector<int>& image_size) {
+    const auto layout = ii->getLayout();
+    size_t to_hw_shift = 0;
+    switch (layout) {
+        case InferenceEngine::NHWC:
+        {
```

Comment:
you don't need `{`/`}` as you don't declare local variables here

Reply:
Fixed.